sinonjs/referee
# Remove UMD wrapper and (most of) lodash

This PR removes the UMD wrapper around the source files, and instead provides a `dist/` folder with a UMD wrapped bundled version of `referee`.

Additionally, it removes `lodash` dependency, in favour of using native methods, polyfills and using standalone lodash methods. This should make the library slightly faster to install, and in modern runtimes also faster to use.

Reviewers are advised to read commits in turn, instead of all at once.

#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm run prepublish` (this runs `build`)
4. Observe that there is a new `dist/` folder, with a UMD wrapped version
5. If you're still here, try using the `dist` version in a project
